src/main/scala/dsptools/Chisel3TestersCompatibility.scala

```diff
@@ -0,0 +1,14 @@
+package chisel3.iotesters
```

Comment:
Anyways, quick turnout to me = ~3 days max. Otherwise, I'll prob merge this with master, unless someone else has other complaints? It can be fixed later. It's not like this repo doesn't need a lot of refactoring to begin with... 

Maybe a better question to pose to @ducky64 is -- why privatize any of the stuff in my Compatibility file? I don't think it's possible that using these things will cause Chisel to break in any way. I get why addPin can be dangerous, but most of these things just add more visibility (like how a signal is actually specified in Verilog, etc.) that people can then use for some post processing (or in my case, Verilog TB creation...).